### PR TITLE
Add `StandaloneCallToActionListBlock`

### PR DESCRIPTION
**Pull request by SebiVPS**
Add `StandaloneCallToActionListBlock` which contains alignment.
We discussed that we need to align the call to action list block items.

### CODE REVIEW

File: site/src/common/blocks/CallToActionListBlock.tsx [hunk 1]
```diff
@@ -17,14 +16,6 @@ export const CallToActionListBlock = withPreview(
...
 );
 
-export const PageContentCallToActionListBlock = (props: CallToActionListBlockProps) => (
-    <PageLayout grid>
-        <PageLayoutContent>
-            <CallToActionListBlock {...props} />
-        </PageLayoutContent>
-    </PageLayout>
-);
-
```
Comment (johnnyomair) [inline site/src/common/blocks/CallToActionListBlock.tsx:27]: Great that you've thought about removing this now unneeded code 👏🏼

File: site/src/common/blocks/StandaloneCallToActionListBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <PageLayout grid>
+        <PageLayoutContent>
+            <StandaloneCallToActionListBlock {...props} />
+        </PageLayoutContent>
+    </PageLayout>
+);
+
+const Root = styled.div<{ $alignment: CSSProperties["justifyContent"] }>`
+    display: flex;
+    justify-content: ${({ $alignment }) => $alignment};
```
Comment (johnnyomair) [inline site/src/common/blocks/StandaloneCallToActionListBlock.tsx:32]: Shouldn't this be flex-start and flex-end instead of start and end?

Comment (SebiVPS) [reply site/src/common/blocks/StandaloneCallToActionListBlock.tsx:38]: interesting, on MDN Web Docs it says start, end and left, right are possible values.
on w3schools with flex-start and flex-end, 
it all does work.

I added a map for flex-start and flex-end values.

Comment (johnnyomair) [reply site/src/common/blocks/StandaloneCallToActionListBlock.tsx:38]: @jamesricky what's your opinion? IMO we can implement it without mapping if start and end are valid. 

All my CSS knowledge is based around 2017 😁

Comment (jamesricky) [reply site/src/common/blocks/StandaloneCallToActionListBlock.tsx:38]: I'd prefer `flex-start`/`flex-end` as it seems to have slightly better browser support. 
But I'm fine with both 😄

File: admin/src/common/blocks/StandaloneCallToActionListBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            callToActionList: {
+                block: CallToActionListBlock,
+            },
+            alignment: {
+                block: createCompositeBlockSelectField<StandaloneCallToActionListBlockData["alignment"]>({
+                    defaultValue: "left",
+                    options: [
+                        { value: "left", label: <FormattedMessage id="standaloneCallToActionList.alignment.left" defaultMessage="left" /> },
+                        { value: "center", label: <FormattedMessage id="standaloneCallToActionList.alignment.center" defaultMessage="center" /> },
+                        { value: "right", label: <FormattedMessage id="standaloneCallToActionList.alignment.right" defaultMessage="right" /> },
```
Comment (jamesricky) [inline admin/src/common/blocks/StandaloneCallToActionListBlock.tsx:20]: Probably out of scope for this PR, but should we add arrays for `leftRightOptions` and `leftRightCenterOptions` to some kind of common/utils file, as this is quite often needed? 🤔 

Then we also wouldn't need unique message IDs which would make translation slightly easier.

Comment (johnnyomair) [reply admin/src/common/blocks/StandaloneCallToActionListBlock.tsx:20]: The messages should probably be defined in [messages.ts](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/messages.ts)

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**